geojson_pydantic/features.py

```diff
@@ -38,6 +39,16 @@ def __geo_interface__(self):
         """GeoJSON-like protocol for geo-spatial (GIS) vector data."""
         return self.dict()
 
+    @classmethod
+    def validate(cls, value):
+        """Validate input."""
+        try:
+            value = json.loads(value)
+        except TypeError:
+            pass
```

Comment:
Why not validation of `BaseModel` objects?

```suggestion
            try:
                return cls(**value.dict())
            except (AttributeError, ValidationError):
                pass
```